cwms-data-api/src/main/java/cwms/cda/data/dto/TimeZone.java

```diff
@@ -20,30 +21,26 @@
 @FormattableWith(contentType = Formats.JSONV2, formatter = JsonV2.class, aliases = {Formats.DEFAULT, Formats.JSON})
 @FormattableWith(contentType = Formats.XMLV2, formatter = XMLv2.class, aliases = {Formats.XML})
 @JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
-public final class TimeZone extends CwmsDTO
-{
-	private String timeZone;
+public final class TimeZone extends CwmsDTOBase {
```

Comment:
Recommend changing this class name to be TimeZoneName or TimeZoneId to clarify it as not having TimeZone business rules and separate it from the java class sharing the same name.

Reply:
Renamed to `TimeZoneId` in: https://github.com/USACE/cwms-data-api/pull/787